pkg/report/report.go

```diff
@@ -251,6 +251,11 @@ type replacement struct {
 
 func replaceTable(replacements []replacement, str string) string {
 	for _, repl := range replacements {
+		for stop := false; !stop; {
```

Comment:
Does any test fail if we remove this loop? I don't immediately see any of these [X:Y] patterns in the changed tests.
If would be good to have tests for this behavior.

Reply:
Indeed. Added such a test and also removed one extra replacement call in this function.